Formula/kore.rb

```diff
@@ -30,9 +30,6 @@ def install
     # Current Makefile hardcodes paths for default MacPorts/Homebrew.
     ENV.prepend "CFLAGS", "-I#{Formula["openssl@1.1"].opt_include}"
     ENV.prepend "LDFLAGS", "-L#{Formula["openssl@1.1"].opt_lib}"
```

Comment:
Upstream now uses `pkg-config` rather than hardcoded paths, so these may also be unnecessary https://github.com/jorisvink/kore/commit/995b6b8586a6ec44884f6690355baf1b79a1ab71